x-pack/plugins/lens/public/editor_frame_service/editor_frame/frame_layout.scss

```diff
@@ -28,7 +28,8 @@
   // Leave out left padding so the left sidebar's focus states are visible outside of content bounds
   // This also means needing to add same amount of margin to page content and suggestion items
   padding: $euiSize $euiSize 0;
-
+  position: relative;
+  z-index: $euiZLevel1;
```

Comment:
@mbondyra , This is causing some major conflicts with the global navigation. And really anything above a level 1 will. Can I suggest taking stock of all the z-indexes in Lens and consider making your own variables starting at like `z-index: 2`. This evaluates to `1000`.

Reply:
Oh here's the screenshot.

<img width="1279" alt="Screen Shot 2021-04-01 at 16 19 13 PM" src="https://user-images.githubusercontent.com/549577/113349599-4f08dd80-9306-11eb-9922-55b0df863910.png">

Reply:
Hey @cchaos, thanks for letting me know, PR with correction coming in!